easybuild/easyconfigs/g/Go/Go-1.11.5.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'Tarball'
+
+name = 'Go'
+version = '1.11.5'
+
+homepage = 'http://www.golang.org'
+description = """Go is an open source programming language that makes it easy to build
+ simple, reliable, and efficient software."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://dl.google.com/go']
+sources = ['%(namelower)s%(version)s.linux-amd64.tar.gz']
+checksums = ['ff54aafedff961eb94792487e827515da683d61a5f9482f668008832631e5d25']
+
+sanity_check_paths = {
+    'files': ['bin/go', 'bin/gofmt'],
+    'dirs': ['api', 'doc', 'lib', 'pkg'],
+}
+
+postinstallcmds = [' mv %(installdir)s/go/* %(installdir)s ']
```

Comment:
@HPC-UniOldenburg Can you explain why this is needed exactly?

Reply:
@boegel 
go seems to get installed in a subdirectory within the installdir called 'go'.
(So that the path to the bin folder is $EBROOTGO/go/bin)
This leads to  the result that the bin folder doesn't get recognised by the systems environment variable.
But I just noticed, that the option _'start_dir = 'go'_ is a way more elegant way to solve this.